CanDIG/candigv2-ingest
# New API + schema for htsget/katsu ingest

- Addresses DIG-1222, DIG-1253
- Adds an OpenAPI schema for Katsu ingest
- Changes Katsu ingest endpoint to /ingest/clinical_donors & uses connexion/openAPI schema for validation
- Implements endpoint for adding S3 credentials to vault
- Implements new htsget ingest model at /ingest/moh_variants/{program ID} endpoint

Testing:
* Try Katsu ingest endpoint (same instructions as before)
* Add an S3 credential to vault and ingest genomic data using the new request format (see README.md for instructions)
